step-certificates/values.yaml

```diff
@@ -108,6 +108,15 @@ inject:
         }
 
   certificates:
+    # certificate_issuer_crt contains the text of the Certificate Issuer certificate in PEM format.
```

Comment:
The name of the property is `certificate_issuer`

Reply:
fixed